app/controllers/concerns/signature_verification.rb

```diff
@@ -242,7 +242,7 @@ def stoplight_wrapper
     Stoplight("source:#{request.remote_ip}")
       .with_threshold(1)
       .with_cool_off_time(5.minutes.seconds)
-      .with_error_handler { |error, handle| error.is_a?(HTTP::Error) || error.is_a?(OpenSSL::SSL::SSLError) ? handle.call(error) : raise(error) }
+      .with_error_handler { |error, handle| Mastodon::HTTP_CONNECTION_ERRORS.any? { |error_class| error.is_a?(error_class) } ? handle.call(error) : raise(error) }
```

Comment:
This feels especially relevant in light of today's announcment of M4 Mac Mini computers. I bet they could run this comparison all day long on their performance cores.

Reply:
Just think about how a just-announced MacBook Pro now featuring a FOURTY (40) CORE GPU would blow through this thing. Yikes. Get. Out. Of. The. Way.